### PR TITLE
Add armenian locale

### PR DESCRIPTION
**Pull request by hovikman**
The following providers have been added for armenian language:
- address
- color
- company
- date_time
- job
- lorem
- person
- phone_number

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -135,8 +135,8 @@ Included localized providers:
 -  `ar\_EG <https://faker.readthedocs.io/en/master/locales/ar_EG.html>`__ - Arabic (Egypt)
 -  `ar\_PS <https://faker.readthedocs.io/en/master/locales/ar_PS.html>`__ - Arabic (Palestine)
 -  `ar\_SA <https://faker.readthedocs.io/en/master/locales/ar_SA.html>`__ - Arabic (Saudi Arabia)
--  `bs\_BA <https://faker.readthedocs.io/en/master/locales/bs_BA.html>`__ - Bosnian
 -  `bg\_BG <https://faker.readthedocs.io/en/master/locales/bg_BG.html>`__ - Bulgarian
+-  `bs\_BA <https://faker.readthedocs.io/en/master/locales/bs_BA.html>`__ - Bosnian
```
Comment (fcurella) [inline README.rst]: we keep this sorted alphabetically by the their verbose nome, not their locale code

File: faker/providers/person/hy_AM/__init__.py [hunk 2]
```diff
@@ -0,0 +1,985 @@
...
+        '{{first_name_male}} {{last_name}}',
+    )
+
+    formats_female = (
+        '{{first_name_female}} {{last_name}}',
+    )
+
+    formats = formats_male + formats_female
+
+    first_names_male = (
```
Comment (fcurella) [inline faker/providers/person/hy_AM/__init__.py]: could you include the source for this dataset?

File: tests/providers/test_person.py [hunk 3]
```diff
@@ -266,7 +267,7 @@ def test_last_name(self):
         # All last names apply for all genders.
         assert hasattr(ZhCNProvider, 'last_names')
 
-        # General first name.
+        # General last name.
```
Comment (fcurella) [inline tests/providers/test_person.py]: Nice catch. Thank you!

File: README.rst (hunk 1 above)
Comment (hovikman) [reply README.rst]: but the list is not ordered by their verbose name either, for example Chinese at the end of the list, Dutch is in the middle.

File: tests/providers/test_person.py (hunk 3 above)
Comment (hovikman) [reply tests/providers/test_person.py]: You are very welcome.

File: README.rst (hunk 1 above)
Comment (fcurella) [reply README.rst]: Sorry, my mistake. You're correct :)

File: faker/providers/person/hy_AM/__init__.py (hunk 2 above)
Comment (hovikman) [reply faker/providers/person/hy_AM/__init__.py]: I've add the source for name datasets, thank you for point this out.

**Comment by fcurella** [comment]
Thank you! ✨

**Pull request merged**